### DISCUSSION TITLE
Use full suffix for file filter options.

### DISCUSSION THREAD

**Discussion by adamisproductiv** [community/community#12236] [Pull Requests]
In a PR that has `*.js` and `*.test.js` I'd like filter options to see them separately.

I'd like to review code by looking at the non-tests first and then the tests, however currently that is challenging as I can only filter to `*.js` which includes all files. If the options were the full suffix and not just the final piece my workflow would work.

**Comment by villelahdenvuo** [comment]
Same for `*.spec.{js,ts}`, a common pattern.
